dappnode_package.json

```diff
@@ -8,7 +8,11 @@
   "description": "Validate with prysm: a Go implementation of the Ethereum 2.0 Serenity protocol and open source project created by Prysmatic Labs. Beacon node which powers the beacon chain at the core of Ethereum 2.0\n\nIt includes a Grafana dashboard for the [DMS](http://my.dappnode/#/installer/dms.dnp.dappnode.eth) thanks to the amazing work of [metanull-operator](https://github.com/metanull-operator/eth2-grafana)",
   "type": "service",
   "architectures": ["linux/amd64"],
-  "chain": "ethereum2-beacon-chain-prysm",
+  "chain": {
+    "driver": "ethereum",
```

Comment:
Must be `ethereum-beacon-chain`

https://github.com/dappnode/DNP_DAPPMANAGER/blob/14f56e8ec1726b068c3c79a8da23673c484ae9f6/packages/dappmanager/src/modules/chains/drivers/index.ts#L34